pkg/operator/ceph/object/notification/controller.go

```diff
@@ -128,56 +134,69 @@ func (r *ReconcileNotifications) reconcile(request reconcile.Request) (reconcile
 		return reconcile.Result{}, nil
 	}
 
+	namespace := notification.Namespace
+	bnName := types.NamespacedName{Namespace: namespace, Name: notification.Name}
+	r.recorder.Eventf(notification, kapiv1.EventTypeNormal, string(cephv1.ReconcileStarted), "Start reconciling %q", bnName)
 	// get the topic associated with the notification, and make sure it is provisioned
 	topicName := types.NamespacedName{Namespace: notification.Namespace, Name: notification.Spec.Topic}
 	bucketTopic, err := topic.GetProvisioned(r.client, r.opManagerContext, topicName)
 	if err != nil {
-		logger.Infof("CephBucketTopic %q not provisioned yet", topicName)
+		failureMessage := fmt.Sprintf("Reconcile failed for %s, topic %q not provisioned yet", bnName, topicName)
+		logger.Info(failureMessage)
+		r.recorder.Event(notification, kapiv1.EventTypeNormal, string(cephv1.ReconcileFailed), failureMessage)
```

Comment:
Failure events are handled automatically by reading the error message when `ReportReconcileResult` is used.